content/guides/tutorial-laravel.md

```diff
@@ -130,6 +130,21 @@ If one of your scheduled tasks needs to allocate more memory than this limit, th
 To allow it to use more memory, you can call [`ini_set()`](https://www.php.net/manual/en/function.ini-set) inside a `php_sapi_name() === 'cli'` condition from an early hook to the app's lifecycle (like the `AppServiceProvider`).
 See [this Gist](https://gist.github.com/dsferruzza/e57dd3db957efe7a649325868f0024a4) for an example implementation.
 
+### Optional: Configure trusted proxy
+
+To ensure Laravel correctly handles HTTP requests when using the Clever Cloud HTTP reverse proxy (Sōzu), add the following code to the `config/trustedproxy.php` file:
+
+```php
+<?php
+
+return [
+    'proxies' => '*',
```

Comment:
```suggestion
    'proxies' => env('CC_REVERSE_PROXY_IPS', ''),
```

I have been using this for multiple years. But I guess using `*` is not a big security issue as you can't reach a Clever Cloud VM directly(?)